vagrant/lib/forklift/box_distributor.rb

```diff
@@ -232,17 +232,9 @@ def configure_libvirt(machine, box, networks = [])
         p.memory = box.fetch('memory').to_i * @settings['scale_memory'].to_i if box.fetch('memory', false)
         p.machine_virtual_size = box.fetch('disk_size') if box.fetch('disk_size', false)
         p.management_network_domain = create_domain(box) if p.respond_to?(:management_network_domain)
+        p.qemu_use_session = @settings.fetch('libvirt_qemu_use_session', true)
```

Comment:
My point was that not every system is using sessions by default, which is why I added the trailing `if ...` so we could have an undefined default. This will break on distros where the session is not set up by default.